contrib/bash_compl/_rgbasm.bash

```diff
@@ -0,0 +1,65 @@
+
+# known bugs - no support for newlines in filenames
+
+
+# reference: https://iridakos.com/programming/2018/03/01/bash-programmable-completion-tutorial
+
+_rgbasm_completions() {
+    local words_len=${#COMP_WORDS[@]} # the # counts the len, @ means all members
+    local asm_files=( $(find -maxdepth 1 -type f -iname "*.inc" -o -iname "*.z80" -o -iname "*.asm" -o -iname "*.sm83"|sed -E "s_^./__") )
+    # using *.z80 -> if no .z80 files are found it uses literal "*.z80"
+    # using ./*.z80 would show options starting with an ugly ./
+    # this is a hard decision... maybe use sed to filter ./*?
```

Comment:
```suggestion
```

(This comment appears outdated, since the above line does use `sed -E "s_^./__`.)

Reply:
I was trying to say "this is an ugly solution. but alternatives are equally ugly" you can remove it if you think it doesn't matter